### DISCUSSION TITLE
Exception django.urls.exceptions.NoReverseMatch

### DISCUSSION THREAD

**Discussion by lnxcode** [netbox-community/netbox#15503] [Help Wanted!]
I have some exception below:

```
Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'django.urls.exceptions.NoReverseMatch'>

Reverse for 'number_list' not found. 'number_list' is not a valid view function or pattern name.

Python version: 3.11.4
NetBox version: 3.7.2
Plugins: 
  netbox_bgp: 0.12.0
  netbox_secrets: 1.10.2
  phonebox_plugin: v0.0.7-beta.2
```

How can I fix this? 
Thank you.

**Comment by candlerb** [comment]
Probably by disabling one of the plugins. "number_list" sounds like it might belong to the "phonebox_plugin".  If it hasn't been updated to work with Netbox v3.7, then it will break.

**Comment by jeremystretch** [comment]
Try enabling debug (set `DEBUG=True` in `configuration.py`) to get the full stack trace, which should tell you where the error is occurring.

**Comment by lnxcode** [comment]
Thank you. The problem was in phonebox_plugin.
